api4/cloud.go

```diff
@@ -83,7 +83,7 @@ func getSubscriptionStats(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	count, err := c.App.Srv().Store.User().Count(model.UserCountOptions{})
+	count, err := c.App.GetTotalUsersCount()
 	if err != nil {
 		c.Err = model.NewAppError("Api4.getSubscriptionStats", "app.user.get_total_users_count.app_error", nil, err.Error(), http.StatusInternalServerError)
```

Comment:
I think I would prefer to create this `AppError` from the new created app function, and here just use the `c.Err = err`.

Reply:
@jespino  if i create this error in the app layer `where` param of `NewAppError`, should be "GetTotalUsersCount". is it okay? 
I'm new to this project and I don't exactly know how does error handling work. 

so I change `GetTotalUsersCount` to: 

```go
func (a *App) GetTotalUsersCount() (int64, error) {
  count, err := a.Srv().Store.User().Count(model.UserCountOptions{})
  if err != nil {
    return nil, model.NewAppError("GetTotalUsersCount", "app.user.get_total_users_count.app_error", nil, err.Error(), http.StatusInternalServerError)
  }
}
```

and then I call that like this: 

```go
count, err := c.App.GetTotalUsersCount()
if err != nil {
    c.Err = err
}
```

let me know if its okay.